frontend/src/__tests__/cypress/cypress/pages/projects.ts

```diff
@@ -82,6 +82,18 @@ class CreateEditProjectModal extends Modal {
   }
 }
 
+class DataConnectionRow extends TableRow {
+  findWorkbenchConnection() {
+    return this.find().find(`[data-label="Connected workbenches"]`);
+  }
+}
+
+class DataConnectionDeleteModal extends DeleteModal {
+  findSubmitButton() {
+    return this.findFooter().findByRole('button', { name: 'Delete data connection' });
+  }
```

Comment:
Is this necessary? The default should already find the submit button if it contains the word `Delete`.